### DISCUSSION TITLE
Create GitHub Action that converts issue to discussion and comments

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#20400] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

How about having a label and action like `should-be-a-discussion` which:
- converts the issue to a discussion
- adds a comment to the discussion saying: “A maintainer has converted this to a discussion. Maybe next time start with a discussion?” etc.

When the issue is converted to a discussion we can still access to the labels, so we could:

- read the label to determine which comment to make (invalid bug report, poorly thought through feature request, etc.)
- remove the labels after

Technical things to figure out before we start work:

- [ ] Is there an API to convert issues to discussions?
- [ ] Does the `GITHUB_TOKEN` in Action have permissions to call the API we need?

**Comment by rarkins** [comment]
I don't know if GitHub's API supports converting issues to discussions. @evi-liu can you confirm?

**Comment by viceice** [comment]
would be very nice as we currently can't do this via the mobile apps

**Comment by HonkingGoose** [comment]
Resolved by this label action config:

https://github.com/renovatebot/renovate/blob/3c9cb362490c7cc3430f94260c43524419abe1ff/.github/label-actions.yml#L169-L185
